install_files/securedrop-ossec-agent/DEBIAN/control.j2

```diff
@@ -0,0 +1,18 @@
+Source: ossec.net
+Section: web
+Priority: optional
+Maintainer: SecureDrop Team <securedrop@freedom.press>
+Homepage: https://securedrop.org
+Package: securedrop-ossec-agent
+Version: 3.6.0+1.6.0~rc1
+Architecture: amd64
+{% if securedrop_build_focal_support %}
+Depends: libevent-2.1.7,libpcre2-8-0,ossec-agent,securedrop-keyring,securedrop-config
+{% else %}
+Depends: libevent-1.4-2,libpcre2-8-0,ossec-agent,securedrop-keyring,securedrop-config
+{% endif %}
+Replaces: ossec-agent
+Conflicts: securedrop-ossec-server
+Description: Installs the securedrop pre-configured OSSEC agent
+  This package installs an OSSEC agent pre-configured for the
+  SecureDrop app server.
```

Comment:
Solid approach to multi-distro support. :+1: However, elsewhere in the diff I see that ` install_files/securedrop-ossec-agent/DEBIAN/control → ...s/build-ossec-deb-pkg/files/control-focal. Given the `control.j2` introduction here, it apepars that `build-ossec-deb-pkg/files/control-focal` is no longer necessary, and should be removed.